src/app/lib/loop-http/index.ts

```diff
@@ -0,0 +1,126 @@
+import { stringify } from 'query-string';
+import { NetworkError } from './errors';
+import { parseLoopErrorResponse } from './utils';
+import * as T from './types';
+
+export type ApiMethod = 'GET' | 'POST' | 'PUT' | 'DELETE';
+
+export class LoopHttpClient {
+  url: string;
+
+  constructor(url: string) {
+    // Remove trailing slash for consistency
+    this.url = url.replace(/\/$/, '');
+  }
+
+  // Public API methods
+
+  getLoopOutTerms = () => {
+    return this.request<T.GetLoopTermsResponse>('GET', `/v1/loop/out/terms`);
+  };
+
+  getLoopInTerms = () => {
+    return this.request<T.GetLoopTermsResponse>('GET', `/v1/loop/in/terms`);
+  };
+
+  getLoopOutQuote = (amt: string, conf: string) => {
+    return this.request<T.GetLoopQuoteResponse>(
+      'GET',
+      `/v1/loop/out/quote/${amt}?conf_target=${conf}`,
+      undefined,
```

Comment:
You should pass `{ conf_target: conf }` as the third arg here instead of doing interpolation and undefined.

Reply:
updated third arg